src/layouts/docs/docs-layout.module.css

```diff
@@ -25,31 +25,261 @@
   height: 100%;
   justify-content: center;
   overflow-y: auto;
-  padding: 24px 24px 100px 24px;
+  padding-bottom: 100px;
+  padding-left: 48px;
+  padding-right: 24px;
+  padding-top: 32px;
```

Comment:
Is there a reason we expanded this to the long-form syntax?

Reply:
I did it for readability. Personally, it's easier for me to parse this mentally than it is the shorthand!

Reply:
☝️ Will +1 that sometimes I find the shorthand higher effort than separate properties.

I am all for consistency though... one related thought here is whether we should adopt something like https://github.com/hudochenkov/postcss-sorting. This way we could have clearly encoded, and automatically applied, conventions for things like this (much like we have for JS already!). Also wary that such a plugin would be a great way to just bikeshed for a while... but feels like CSS is kind of a second class citizen in terms of linting & auto-formatting of CSS properties. Will add this as a task for discussion on the platform board!

Reply:
I'm working on an agenda-like thing for a code style jam session (just adding links for things that are relevant, not working too hard on it since there are endless links haha), I'll add this to that list too!